apps/web/src/components/Menu/MenuItem.tsx

```diff
@@ -0,0 +1,54 @@
+import { Flex, Icon, Text } from "@chakra-ui/react";
+
+import { ChevronRightIcon } from "../../assets/icons";
+import { useColor } from "../../styles/useColor";
+
+export type TMenuItem = {
```

Comment:
I definitely saw it [somewhere](https://github.com/trilitech/umami-v2/pull/1685/files#diff-d4717dbfead9ae5b97ceefdb98f6c9288aa208b2a53bed2b891dcfe874f9ae8dR1)